app/Ticket.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace App;
+use Illuminate\Database\Eloquent\Model;
+
+class Ticket extends Model
+{	
+	protected $guard = ['id'];
+	
+	public function comments()
+	{
+		return $this->hasMany('App\Comment', 'post_id');
```

Comment:
```suggestion
		return $this->hasMany(Comment::class, 'post_id');
```
Sử dụng static class thay vì string